DataProducts/test/lassipatanen/Job/JobPosting.json

```diff
@@ -0,0 +1,201 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Job Posting",
+    "description": "Data product for Job Posting",
+    "version": "1.0"
+  },
+  "paths": {
+    "/test/Job/JobPosting": {
+      "post": {
+        "summary": "test/Job/JobPosting",
+        "operationId": "request_test_JobPosting",
+        "parameters": [
+          {
+            "description": "Optional consent token",
+            "required": false,
+            "schema": {
+              "title": "X-Consent-Token",
+              "type": "string",
+              "description": "Optional consent token"
+            },
+            "name": "x-consent-token",
+            "in": "header"
+          },
+          {
+            "description": "The login token. Value should be \"Bearer [token]\"",
+            "required": false,
+            "schema": {
+              "title": "Authorization",
+              "type": "string",
+              "description": "The login token. Value should be \"Bearer [token]\""
+            },
+            "name": "authorization",
+            "in": "header"
+          },
+          {
+            "description": "The bare domain of the system that provided the token.",
+            "required": false,
+            "schema": {
+              "title": "X-Authorization-Provider",
+              "type": "string",
+              "description": "The bare domain of the system that provided the token."
+            },
+            "name": "x-authorization-provider",
+            "in": "header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/JobPostingResponse"
+                }
+              }
+            }
+          },
+          "422": {
+            "description": "Validation Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/HTTPValidationError"
+                }
+              }
+            }
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/JobPostingRequest"
+              }
+            }
+          }
+        },
+        "description": "Job Posting"
+      },
+      "parameters": []
```

Comment:
The validation fails due to this with the `OnlyPostMethodAllowed`. The OpenAPI Spec does indeed support the `parameters`, but the limited subset of it that's used here does not. Considering you don't have anything defined within it, easiest is probably just to remove it rather than checking if the tooling could be modified to support it.

Reply:
I will remove parameters from here.